src/modules/daemonconfiguration/src/daemonconfiguration.rs

```diff
@@ -113,71 +167,161 @@ impl SystemctlInfo for Systemctl {
         })
     }
 
-    fn show_substate_property(name: &str) -> Result<String, MmiError> {
+    fn show_substate_property(&self, name: &str) -> Result<String> {
         let output = {
             Command::new("systemctl")
                 .args(["show", name, "--property=SubState"])
                 .output()?
         };
         Ok(String::from(std::str::from_utf8(&output.stdout)?))
     }
+
+    fn exists(&self, name: &str) -> Result<bool> {
+        Ok(systemctl_crate::exists(name)?)
+    }
+
+    fn start(&mut self, name: &str) -> Result<bool> {
+        let unit = systemctl::Unit::from_systemctl(name)?;
+        if unit.is_active()? {
+            // Unit is already started
+            Ok(false)
+        } else {
```

Comment:
> } else { [](http://example.com/codeflow?start=7&length=9)

}
else
{

... here and elsewhere (always break new line for block start opened by {)

Reply:
See above comment